sdk/bsc-api.js

```diff
@@ -25,7 +25,7 @@ const BSC_WEB3 = new Eth(BSC_RPC_URL);
 const BSC_SCAN = new Etherscan(BSC_SCAN_KEY, "https://api.bscscan.com/api");
 const BSC_LIMITER = new Bottleneck({ maxConcurrent: 10, minTime: 50 });
 const BSC_MULTICALL_PROVIDER = "0xe7144e57d832c9005D252f415d205b4b8D78228e";
-const BSC_GET_LOGS_BATCH_SIZE = 5000;
```

Comment:
Having such a large batch size causing network errors when fetching many logs for pancakeswap or bakeryswap

Reply:
500 is too small and it will make fetch taking much longer time. I think the problem is related to internal tx fetching, see comments below